src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -416,14 +416,46 @@
 
 helps['functionapp plan create'] = """
 type: command
-short-summary: Create an App Service Plan for an Azure Function
+short-summary: Create an App Service Plan for an Azure Function.
 examples:
+  - name: Create an elastic premium app service plan with burst out capability up to 10 instances.
+    text: >
+        az functionapp plan create -g MyResourceGroup -n MyPlan --min-instances 1 --max-burst 10 --sku EP1
   - name: Create a basic app service plan.
     text: >
         az functionapp plan create -g MyResourceGroup -n MyPlan --sku B1
-  - name: Create a standard app service plan with with four workers.
+"""
+
+helps['functionapp plan update'] = """
+type: command
+short-summary: Update an App Service plan for an Azure Function.
+examples:
+  - name: Update an app service plan to EP2 sku with twenty maximum workers.
+    text: >
+        az functionapp plan update -g MyResourceGroup -n MyPlan --max-burst 20 --sku EP2
+"""
+
+helps['functionapp plan delete'] = """
+type: command
+short-summary: Delete an App Service Plan.
+"""
+
+helps['functionapp plan list'] = """
+type: command
+short-summary: List App Service Plans.
+examples:
+  - name: List all Elastic Premium 1 tier App Service plans.
     text: >
-        az functionapp plan create -g MyResourceGroup -n MyPlan --number-of-workers 4 --sku S1
+        az functionapp plan list --query "[?sku.tier=='EP1']"
+"""
+
+helps['functionapp plan show'] = """
+type: command
+short-summary: Get the App Service Plans for a resource group or a set of resource groups.
+examples:
+  - name: Get the app service plans for a resource group or a set of resource groups. (autogenerated)
+    text: az functionapp plan show --name MyAppServicePlan   --resource-group MyResourceGroup
```

Comment:
Please take care of the extra white-spaces here in the next PR. I am merging it since we are running out of the time for the current release